blade-graphics/src/gles/resource.rs

```diff
@@ -184,13 +201,27 @@ impl crate::traits::ResourceDevice for super::Context {
                         );
                     }
                     crate::TextureDimension::D2 => {
-                        gl.tex_storage_2d(
-                            target,
-                            desc.mip_level_count as i32,
-                            format_desc.internal,
-                            desc.size.width as i32,
-                            desc.size.height as i32,
-                        );
+                        if desc.sample_count <= 1 {
+                            gl.tex_storage_2d(
+                                target,
+                                desc.mip_level_count as i32,
+                                format_desc.internal,
+                                desc.size.width as i32,
+                                desc.size.height as i32,
+                            );
+                        } else {
+                            /*
+                                TODO(ErikWDev): How to set mip count and sample count? Not possible in gles?
```

Comment:
I don't think MSAA makes sense with mipmapping in general. We don't need to support this case - can just assert that the mip count is 1